### ISSUE TITLE
ShareThis component needs a button to be accessible

### ISSUE THREAD

**Issue by danlaush** [dpc-sdp/ripple#297]
The default settings of the `vue-social-sharing` plugin renders a `<span>` for each network. This should be an interactable element for accessibility.

I have a fix, will update when I submit the PR.

**Comment by danlaush** [comment]
Closed, fixed in #293
